Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:6.9.1
+
+# Packages required to run the app
+RUN npm install -g foreman
```

Comment:
The less RUN commands there are, the better. Every command creates another one fs layer with its own uuid. Take a look how docker files are structured for most packages:

https://github.com/nodejs/docker-node/blob/d20d305f0bf5935385a32558501f3a5c65e34878/6.9/Dockerfile#L23
https://github.com/docker-library/php/blob/b208b3bd750feea2c236af0586e544dd1daad8a2/7.0/Dockerfile#L41

They have as many commands separated by semicolon in one RUN as possible.